ui/src/components/common/SummaryPageLayout/partials/SummaryStatuses/index.tsx

```diff
@@ -40,6 +43,22 @@ export function SummaryStatuses({
         justifyContent: "center",
       }}
     >
+      {tooltip && (
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "right",
+            width: "100%",
+            justifyContent: "flex-end",
+            paddingRight: "0.2rem",
+            marginBottom: "-1rem",
```

Comment:
Is there anyway we can avoid negative margins?

Reply:
removed -ve margin and refactored it